addons/mail/static/src/components/chatter_topbar/chatter_topbar.js

```diff
@@ -101,6 +101,7 @@ class ChatterTopbar extends Component {
                 on_close: () => {
                     this.chatter.thread.refreshActivities();
                     this.chatter.thread.refresh();
+                    this.trigger('reload');
```

Comment:
Triggering `reload` will likely already refresh thread, so keeping the 2 existing lines might be overkill. To double check to ensure all the RPC (and there are many already with those refresh) are not done 2 times each immediately.